concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -0,0 +1,342 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Luangthongkum-2019-341-1	1	all, all gone, whole		
+Luangthongkum-2019-341-2	2	ant	587	ANT
+Luangthongkum-2019-341-3	3	arrive	1387	ARRIVE
+Luangthongkum-2019-341-4	4	arrow, dart	977	ARROW
+Luangthongkum-2019-341-5	5	ascend, go up, rise	705	GO UP (ASCEND)
+Luangthongkum-2019-341-6	6	ashes, fireplace		
+Luangthongkum-2019-341-7	7	astringent		
+Luangthongkum-2019-341-8	8	bamboo	1927	BAMBOO
+Luangthongkum-2019-341-9	9	bamboo rat		
+Luangthongkum-2019-341-10	10	bamboo strip		
+Luangthongkum-2019-341-11	11	bamboo shoot		
+Luangthongkum-2019-341-12	12	bark (n.), leather, skin	2127	BARK OR SKIN
+Luangthongkum-2019-341-13	13	bat	1793	BAT
+Luangthongkum-2019-341-14	14	bathe	138	BATHE
+Luangthongkum-2019-341-15	15	bear (animal)	1820	BEAR
+Luangthongkum-2019-341-16	16	bedbug	1949	BEDBUG
+Luangthongkum-2019-341-17	17	bee (Apis cerana)		
+Luangthongkum-2019-341-18	18	bee (Apis dorsata)		
+Luangthongkum-2019-341-19	19	believe	1890	BELIEVE
+Luangthongkum-2019-341-20	20	below, under, underneath	1485	BELOW OR UNDER
+Luangthongkum-2019-341-21	21	big, large	1202	BIG
+Luangthongkum-2019-341-22	22	bird	937	BIRD
+Luangthongkum-2019-341-23	23	bite (v.)	1403	BITE
+Luangthongkum-2019-341-24	24	bitter	887	BITTER
+Luangthongkum-2019-341-25	25	blood	946	BLOOD
+Luangthongkum-2019-341-26	26	blow (a fire)	176	BLOW (WITH MOUTH)
+Luangthongkum-2019-341-27	27	boar (wild~)	3154	WILD BOAR
+Luangthongkum-2019-341-28	28	boat	1844	BOAT
+Luangthongkum-2019-341-29	29	body dirt		
+Luangthongkum-2019-341-30	30	boil (v.)	2493	BOIL
+Luangthongkum-2019-341-31	31	bone	1394	BONE
+Luangthongkum-2019-341-32	32	braid		
+Luangthongkum-2019-341-33	33	brain	1583	BRAIN
+Luangthongkum-2019-341-34	34	bran,chaff	2363	CHAFF (HUSK)
```

Comment:
A space could be inserted after the comma (unless it looks like this in the original list).

Reply:
This should be unmapped as we don't yet have a hyperonym for bran and chaff.